src/type_checker/resources/primitives/float.py

```diff
@@ -0,0 +1,46 @@
+class float:
+    def __add__(self, other: int) -> float: pass
+    def __add__(self, other: float) -> float: pass
+    def __add__(self, other: complex) -> complex: pass
+    def __add__(self) -> float: return self
+
+    def __sub__(self, other: int) -> float: pass
+    def __sub__(self, other: float) -> float: pass
+    def __sub__(self, other: complex) -> complex: pass
+    def __sub__(self) -> float: return -self
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The special method '__sub__' expects 1 param(s), 0 was given](https://app.codacy.com/app/JSAbrahams/mamba/pullRequest?prid=4098440)